TYPO3.TYPO3CR/Classes/TYPO3/TYPO3CR/Domain/Model/ExpressionBasedNodeLabelGenerator.php

```diff
@@ -71,13 +74,27 @@ public function initializeObject()
      */
     public function getLabel(NodeInterface $node, $crop = true)
     {
-        $label = \TYPO3\Eel\Utility::evaluateEelExpression($this->getExpression(), $this->eelEvaluator, array('node' => $node), $this->defaultContextConfiguration);
+        $label = null;
+        if ($node->getNodeType()->isOfType('TYPO3.Neos:ContentCollection') && $node->isAutoCreated()) {
```

Comment:
This is a no-go as it introduces a semi dependency of Neos in CR. If we want to introduce this kind of feature it must work for all node types. I could accept that and I can see how nice this can look like, but I am not 100% sold for this yet...